proposed/2021/VisualStudioDeepLinkNuGetPackageInstall.md

```diff
@@ -0,0 +1,66 @@
+Visual Studio Package Install Deep Link Spec
+
+- Status: **In Review**
+- Author(s): [David Cueva Cortez](https://github.com/dcc7497)
+- Issue: [11032](https://github.com/NuGet/Home/issues/11032) Deep Links in NuGet PM in Visual Studio
+- Type: Feature
+
+## Problem Background
+
+Currently, customers have two main ways of installing NuGet packages: directly from the Visual Studio NuGet Package Manager UI (PMUI) and using a CLI command from nuget.org. This new feature allows customers to click on a deep link that will direct them to the package details page of the focus package in the PMUI. It is there that the customer can then install the focus package making installation a 2-step process. This feature allows for much more flexibility when installing packages as the link can be shared between people, can be embedded into documentation, among other things.
+
+## Who are the customers
+
+- Developers who want a quick and easy way to install packages directly from the PMUI with a click of a button.
```

Comment:
Not sure how important the wording is, but it's not clear here that you mean opening PM UI "From nuget.org" or from a URL. In other words, maybe change "button" to be more clear.